Simplenote/Classes/SPAboutViewController.swift

```diff
@@ -215,24 +224,25 @@ extension SPAboutViewController: UITableViewDataSource {
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = UITableViewCell(style: UITableViewCell.CellStyle.subtitle, reuseIdentifier: "reuseIdentifier")
+        let cell = UITableViewCell(style: UITableViewCell.CellStyle.subtitle, reuseIdentifier: nil)
 
         cell.textLabel?.text = Constants.titles[indexPath.row]
         cell.detailTextLabel?.text = Constants.descriptions[indexPath.row]
         
-        cell.textLabel?.textColor = UIColor.white
-        cell.detailTextLabel?.textColor = UIColor.white
-        cell.backgroundColor = UIColor.clear
-        
+        cell.textLabel?.textColor = .white
+        cell.detailTextLabel?.textColor = .simplenoteBlue10Color
+        cell.detailTextLabel?.font = .preferredFont(forTextStyle: .subheadline)
+        cell.backgroundColor = .clear
+
         let bgColorView = UIView()
         bgColorView.backgroundColor = .simplenoteBlue30Color
         cell.selectedBackgroundView = bgColorView
         
-        let arrowAccessoryView = UIImageView(image: UIImage(named: "icon_arrow_top_right"))
-        arrowAccessoryView.tintColor = .white
-        arrowAccessoryView.frame = CGRect(x: 0, y: 0, width: 20, height: 20)
+        let arrowAccessoryView = UIImageView(image: .image(name: .arrowTopRight))
+        arrowAccessoryView.tintColor = .simplenoteBlue10Color
+        arrowAccessoryView.frame = CGRect(x: 0, y: 0, width: 24, height: 24)
```

Comment:
Nipticky!! (Apologies, I know this was already there), would it be possible to use a constant here?